mosquitto/umbrel-app.yaml

```diff
@@ -0,0 +1,28 @@
+manifestVersion: 1.1
+id: mosquitto
+name: Mosquitto
+tagline: An open source MQTT broker
+icon: https://raw.githubusercontent.com/eclipse-mosquitto/mosquitto/refs/heads/master/logo/mosquitto-logo-only.svg
```

Comment:
This entire icon key-value pair can be deleted. We'll host the icon at https://github.com/getumbrel/umbrel-apps-gallery